SConstruct

```diff
@@ -1,4 +1,74 @@
 # -*- python -*-
-from lsst.sconsUtils import scripts
-# Python-only package
-scripts.BasicSConstruct("ci_ap", disableCc=True, noCfgFile=True)
+import os
+from SCons.Script import Environment, GetOption, Default
+from lsst.sconsUtils.utils import libraryLoaderEnvironment
+
+env = Environment(ENV=os.environ)
+env["ENV"]["OMP_NUM_THREADS"] = "1"  # Disable threading
```

Comment:
This is not the only possible source of threading, but it is indeed the main one that I think you care about.

Reply:
Do we have a canonical list of those? It would be good to add as many as might be necessary.

Reply:
Read it and weep (but also reuse?): https://github.com/lsst/utils/blob/main/python/lsst/utils/threads.py#L38-L57

Reply:
Well, if you reuse anything it should be https://github.com/lsst/utils/blob/main/python/lsst/utils/threads.py#L60.

Reply:
Oh, thank you! That's a handy function to remember.